code/js/rcalc-constant-string-utility.js

```diff
@@ -0,0 +1,10 @@
+// adds to a value px, making it work to change atributes set in px
+const SUFFIX_PX = "px";
+// empty string
+const STRING_EMPTY = "";
+// colors
+const COLOR_BLACK = "#000000";
```

Comment:
Again ... colors are styling related.

Reply:
all constants related to styling in 1 js file called rcalc-constant-html-css